app/views/users/show.html.erb

```diff
@@ -1,51 +1,69 @@
-<div class="row">
-  <div class="col-sm-4">
-    <div class="box center dash-profile">
-      <img class="prof-pic-circle" src="http://api.randomuser.me/portraits/med/women/39.jpg">
-      <br><br>
-      <h3>Gwen Fairfax</h3>
-      <h4>Harvard College</h4>
-      <p><span class="glyphicon glyphicon-user" aria-hidden="true"></span>  Student</p>
-      <br><br>
-      <div class="total-points col-centered">
-        <h2 class="vcenter">76</h2>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-sm-3">
+      <div class="box center dash-profile">
+        <img class="prof-pic-circle" src="/assets/person.png">
+        <br><br>
+        <h3><%= current_user.first_name %> <%= current_user.last_name %></h3>
+        <h4><%= current_user.email %>
+        <!-- <h4>Harvard College</h4>
+        <p><span class="glyphicon glyphicon-user" aria-hidden="true"></span>  Student</p> -->
+        <br><br><br>
+        <div class="total-points col-centered">
+          <h2 class="vcenter">0</h2>
```

Comment:
Probably a good idea to have an aria/tooltip for this, since it is just a number without any text to describe what it's for (to a user who's unfamiliar, or to a screenreader).